tools/releasing/create_binary_release.sh

```diff
@@ -44,6 +44,12 @@ else
     SHASUM="sha512sum"
 fi
 
+cd ..
+
+FLINK_DIR=`pwd`
+RELEASE_DIR=${RELEASE_DIR}/tools/releasing/release
```

Comment:
Shouldn't this be `${FLINK_DIR}/tools/releasing/release`?

Reply:
yes!